### PR TITLE
http: simplify CORS, fix content-type bug

### PR DESCRIPTION
**Pull request by adamdecaf**
Issue: https://github.com/moov-io/moov-io/issues/11

### CODE REVIEW

File: http/server.go [hunk 1]
```diff
@@ -101,11 +100,6 @@ func SetAccessControlAllowHeaders(w http.ResponseWriter, r *http.Request) {
 		w.Header().Set("Access-Control-Allow-Methods", "GET,POST,PATCH,DELETE")
 		w.Header().Set("Access-Control-Allow-Headers", "Cookie,X-User-Id,X-Request-Id,Content-Type")
 		w.Header().Set("Access-Control-Allow-Credentials", "true")
-
-		// Don't overwrite content-type
-		if v := w.Header().Get("Content-Type"); v == "" {
-			w.Header().Set("Content-Type", "text/plain")
```
Comment (adamdecaf) [inline http/server.go]: We were setting this to try and stay within the preflight limits (i.e. not to require a preflight request), but that's already broken by sending credentials across domains.

tl;dr there's no point and this caused a bug in rendering ACH file contents

https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS
